<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_recon2Class</class>
 <widget class="QMainWindow" name="qt_recon2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt_recon2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="img_path_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>img_path</string>
    </property>
   </widget>
   <widget class="QLabel" name="scene_path_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="input_scale_spin">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>150</y>
      <width>33</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="out_scale_spin">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>33</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="sample_len_edt">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>132</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>image_scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>320</y>
      <width>76</width>
      <height>24</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>sample_lenth</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>64</width>
      <height>24</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>input_scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>54</width>
      <height>24</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>out_scale</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="img_size_spin">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>151</y>
      <width>33</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="open_model_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenModel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>entire_recon</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_sub">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>701</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_total">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>701</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="mvs_check">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>112</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>mvs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>110</y>
      <width>64</width>
      <height>24</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>plane_thick</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sift_check">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>112</y>
      <width>44</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sift</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="plane_remove_check">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>112</y>
      <width>103</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>remove_floor</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="smvs_check">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>112</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>smvs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="akaze_sift_check">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>112</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>all</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="plane_tolerance_edt">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="akaze_check">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>112</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>akaze</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="default_setting_check">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>190</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>default_setting</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="seperate_recon_check">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>step_recon</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="use_shade_check">
    <property name="geometry">
     <rect>
      <x>679</x>
      <y>638</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>shading</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scene_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>make_scene</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sfm_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>sfm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tex_btn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>tex_recon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dm_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>depth_recon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fss_btn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>surface_recon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openModelBtn">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>271</y>
      <width>97</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>scaling_model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tex_scale_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>scaling_tex_mesh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image_path_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>94</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>image_set_path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scene_path_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>scene_path</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_recon2.qrc"/>
 </resources>
 <connections/>
</ui>
